docs/about/team/the-software-development-team.md

```diff
@@ -6,6 +6,18 @@ sidebar_position: 2
 
 ## Current team
 
+## [Samuel Pelletier-Evraire](www.linkedin.com/in/samuel-pelletier-evraire) - Application mobile hivers 2024
```

Comment:
Only the job title is necessary, it needs to be written in english. I heard @rngadam say that we'd like to avoid the word "mobile", you could use Portable Application